lms/djangoapps/teams/static/teams/js/views/edit_team.js

```diff
@@ -6,15 +6,20 @@
             'gettext',
             'js/views/fields',
             'teams/js/models/team',
+            'common/js/components/utils/view_utils',
             'text!teams/templates/edit-team.underscore'],
-        function (Backbone, _, gettext, FieldViews, TeamModel, editTeamTemplate) {
+        function (Backbone, _, gettext, FieldViews, TeamModel, ViewUtils, editTeamTemplate) {
             return Backbone.View.extend({
 
                 maxTeamNameLength: 255,
                 maxTeamDescriptionLength: 300,
 
+                submitNoDoubleCLick: function(event) {
```

Comment:
I would move this function definition down below in the file, where createOrUpdateTeam is. It's odd to have any functions above the event handling declaration and initialize methods.